package/obs/rmt-server.spec

```
@@ -34,7 +34,7 @@
 %undefine _find_debuginfo_dwz_opts
 
 Name:           rmt-server
-Version:        2.20
+Version:        2.20.1
```

Bump the minor version for next RMT release

```suggestion
Version:        2.21
```